### PR TITLE
updated country access overrides.

### PR DESCRIPTION
**Pull request by gknisely**
# Issue

Updated country access overrides based on the latest version of https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access_restrictions

## Tasklist

 - [x] Test updates with planet conversion
 - [x] Update the [changelog](CHANGELOG.md)

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

**Comment by nilsnolde** [comment]
Thanks for the chore. Should we add a changelog entry?

**Comment by gknisely** [comment]
> Thanks for the chore. Should we add a changelog entry?

Yup on it.  I am also going to convert the planet locally and test the changes.

**Comment by gknisely** [comment]
Spreadsheet used to compare previous restrictions to updated restrictions.  Red indicates a change

[accessupdates.xlsx](https://github.com/valhalla/valhalla/files/13687076/accessupdates.xlsx)

**Comment by gknisely** [comment]
@nilsnolde good to go.  Tested each change with a new planet.  All looked great.

**Review by nilsnolde — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -55,8 +55,9 @@
    * ADDED: "shape" field to matrix response for CostMatrix only [#4432](https://github.com/valhalla/valhalla/pull/4432)
    * CHANGED: `/expansion`: add field `prev_edge_id`, make the GeoJSON features `LineString`s [#4275](https://github.com/valhalla/valhalla/issues/4275)
    * ADDED: --optimize & --log-details to valhalla_run_matrix [#4355](https://github.com/valhalla/valhalla/pull/4334)
-   * ADDED: most access restrictions to /locate response [#4431](https://github.com/valhalla/valhalla/pull/4431
+   * ADDED: most access restrictions to /locate response [#4431](https://github.com/valhalla/valhalla/pull/4431)
```
Comment (nilsnolde) [inline CHANGELOG.md:58]: thanks for the fix! realized that too in another PR

**Pull request merged**